CONTRIBUTING.md

```diff
@@ -137,6 +137,12 @@ Presto committers are defined as [code owners](https://docs.github.com/en/reposi
 
 New committers are approved by majority vote of the TSC ([see TSC charter](https://github.com/prestodb/tsc/blob/master/CHARTER.md)).  To become a committer, reach out to an [existing TSC member](https://github.com/prestodb/tsc#members) and ask for their feedback on your eligibility (see: [How to become a Presto Committer?](https://github.com/prestodb/presto/wiki/How-to-become-a-Presto-committer%3F)).  Note: to expedite the process, consider creating a document that outlines your Github stats, such as the number of reviews, lines of code added, number of PRs, and outlines particularly outstanding code and review contributions.  If the TSC member believes you are eligible, they will submit your nomination to a vote by the TSC, typically in the form of a PR that adds your handle to the `CODEOWNERS` file.  The process is complete once the PR is merged.
 
+Committers are responsible for providing final approval of pull requests in a timely manner.  If the committer is unable to provide timely review, they should assign the pull request to another committer.
+
+The [CODEOWNERS](CODEOWNERS) file should list the backup committers, called the [committers](https://github.com/orgs/prestodb/teams/committers) team, alongside any file grouping, and as a general
+fallback for file groupings that don't exist.  The TSC may add additional backup committers as needed.  The requirement for addition to the backup committers group is two or more code owners responsibilities in the Presto repository
```

Comment:
I think having people with broad committership rights is valuable, so I think we should encourage growing the list of backup committers (while maintaining high expectations of anyone nominated).  2 things I'm thinking about
1. should we require actually being codeowner of two areas or demonstrating that you *could* be codeowner of two areas. It would be nice to be able to condense some of the beaurocracy if you know you plan to nominate someone as a committer after the second codeowner nomination.  If prefered it could be submitted as a two part nomination (e.g. I'd like to nominate <person> to be codeowner of <second area>.  I'd also like to nominate them to be a global committer), and then people could vote on both aspects
2. Could there be a way to add people as committers who don't have a specific second area that they could obviously codeowners for, but maybe they have a core area and scattered contributions in many different areas?  For example, in any particular second area they may only have one or two contributions, but they've made contributions in a variety of areas, and in that way have demonstrated strong skills/knowledge in the Presto code base.